src/socket-handlers/create-char-handler.js

```diff
@@ -0,0 +1,41 @@
+const BaseHandler = require("./base-handler");
+const { ChatModel } = require("../database/modles");
```

Comment:
looks like ChatModel  not exists https://github.com/nNg9999/node-js-30/tree/06-email/src/database/models

Reply:
add ChatModel